infrastructure/constructs/api.py

```diff
@@ -37,7 +37,7 @@ def __init__(  # pylint: disable=too-many-arguments,too-many-locals
             self,
             "api-users-role",
             role_name=Resource.API_USERS_ROLE_NAME.resource_name,
-            assumed_by=principal,
+            assumed_by=aws_iam.OrganizationPrincipal("o-g9kpx6ff4u"),
```

Comment:
Do we need to also replace the other use of `principal`, in the S3 users role? (At which point we can get rid of the `principal` parameter.)

Reply:
Just to clarify.

s3-users role should be able to be assumed by anyone at LINZ. So if you can achieve that through this same mechanism that's all good.